scijava-ops-engine/src/main/java/org/scijava/ops/engine/impl/DefaultOpEnvironment.java

```diff
@@ -85,10 +85,7 @@
 import org.scijava.priority.Priority;
 import org.scijava.struct.FunctionalMethodType;
 import org.scijava.struct.ItemIO;
-import org.scijava.types.DefaultTypeReifier;
-import org.scijava.types.Nil;
-import org.scijava.types.TypeReifier;
-import org.scijava.types.Types;
+import org.scijava.types.*;
```

Comment:
I'm surprised the maven formatter plugin is okay with this. Anyways, ideally we wouldn't do this.